config/routes.rb

```diff
@@ -3,10 +3,10 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
   scope 'v1' do
-    get '/modsulator/version'
+    get '/modsulator/version', to: 'modsulator#version', as: 'version'
     resources :about, only: :index
     resources :spreadsheet, only: :index
-    resources :modsulator, only: %i[create version]
```

Comment:
rails 8 did not like this non-standard REST route mentioned here, so made it work by adjusting above